Detectors/TRD/calibration/macros/CMakeLists.txt

```diff
@@ -32,8 +32,19 @@ o2_add_test_root_macro(readTRDVoltages.C
                                             O2::CCDB
                       LABELS trd COMPILE_ONLY)
 
+o2_add_test_root_macro(manualCalibFit.C
+                      PUBLIC_LINK_LIBRARIES O2::DataFormatsTRD
```

Comment:
I think you need to add `O2::TRDCalibration` here as well. The CI tests are failing with `/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/Detectors/TRD/calibration/macros/manualCalibFit.C:32:10: fatal error: 'TRDCalibration/CalibratorVdExB.h' file not found`